anta/device.py

```diff
@@ -244,14 +248,15 @@ async def collect(self, command: AntaCommand) -> None:
         """
         try:
             commands = []
-            if self._enable_password is not None:
+            if self.enable and self._enable_password is not None:
```

Comment:
So I can provide enable_password without enable and ANTA will silently ignore enable_password.

Reply:
no more silence in https://github.com/arista-netdevops-community/anta/commit/9d55318d4f5fef3aaff7b400857ded06305b5e90